Tasks/AppStoreRelease/app-store-release.ts

```diff
@@ -172,6 +172,11 @@ async function run() {
         let fastlaneVersionToInstall: string;  //defaults to 'LatestVersion'
         if (fastlaneVersionChoice === 'SpecificVersion') {
             fastlaneVersionToInstall = tl.getInput('fastlaneToolsSpecificVersion', true);
+            if (applicationType.toLocaleLowerCase() === 'macos' &&
+                releaseTrack === 'TestFlight' &&
+                fastlaneVersionToInstall.localeCompare('2.193.0', undefined, { numeric: true }) <= 0 ) {
```

Comment:
Could you please use [semver](https://www.npmjs.com/package/semver) package capabilities for this?

Reply:
done.